vision/automl/src/main/java/com/google/cloud/vision/samples/automl/ModelApi.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Google Inc.
+ * Copyright 2018-19 Google LLC
```

Comment:
You can leave the license file as is, only need to update the year on new files and the java linter will throw an error as it expects a certain year and style based on the configuration.